Master npm install fails on osx - no mouse-forward-back support

rabblebook:patchwork rabble$ npm install
npm ERR! code EBADPLATFORM
npm ERR! notsup Unsupported platform for mouse-forward-back@1.0.1: wanted {"os":"linux","arch":"any"} (current: {"os":"darwin","arch":"x64"})
npm ERR! notsup Valid OS:    linux
npm ERR! notsup Valid Arch:  any
npm ERR! notsup Actual OS:   darwin
npm ERR! notsup Actual Arch: x64

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/rabble/.npm/_logs/2019-01-22T23_23_23_012Z-debug.log

--- a/main-window.js
+++ b/main-window.js
@@ -13,7 +13,12 @@ var ref = require('ssb-ref')
 var setupContextMenuAndSpellCheck = require('./lib/context-menu-and-spellcheck')
 var watch = require('mutant/watch')
 var requireStyle = require('require-style')
-var mouseForwardBack = require('mouse-forward-back')
+
+try {
+  var mouseForwardBack = require('mouse-forward-back')
+} catch (er) {
+  mouseForwardBack = false
+}
 
 module.exports = function (config) {
   var sockets = combine(

--- a/main-window.js
+++ b/main-window.js
@@ -16,7 +16,7 @@ var requireStyle = require('require-style')
 
 try {
   var mouseForwardBack = require('mouse-forward-back')
-} catch (er) {
+} catch (err) {
   mouseForwardBack = false
 }
 

--- a/package.json
+++ b/package.json
@@ -48,7 +48,6 @@
     "micro-css": "^2.0.1",
     "moment-timezone": "^0.5.23",
     "multiserver-dht": "^4.0.0",
-    "mouse-forward-back": "^1.0.1",
     "mutant": "github:mmckegg/mutant#intersection-binding-viewport",
     "mutant-pull-reduce": "^1.1.0",
     "node-abi": "^2.5.0",
@@ -100,5 +99,8 @@
   "devDependencies": {
     "colors": "^1.3.2",
     "electron": "^2.0.13"
+  },
+  "optionalDependencies": {
+    "mouse-forward-back": "^1.0.1"
   }
 }

